src/application-constants.ts

```diff
@@ -7,6 +7,11 @@ export const APPLICATION_NAME = 'synopsys-action'
 
 export const SYNOPSYS_BRIDGE_INSTALL_DIRECTORY_KEY = 'synopsys_bridge_install_directory'
 
+export const BRIDGE_DOWNLOAD_URL_KEY = 'bridge_download_url'
+export const SYNOPSYS_BRIDGE_DOWNLOAD_URL_KEY = 'synopsys_bridge_download_url'
+export const BRIDGE_DOWNLOAD_VERSION_KEY = 'bridge_download_version'
+export const SYNOPSYS_BRIDGE_DOWNLOAD_VERSION_KEY = 'synopsys_bridge_download_version'
+
 // Scan Types
 export const COVERITY_KEY = 'coverity'
 export const POLARIS_KEY = 'polaris'
```

Comment:
can we add a comment for the deprecated fields so that it's easy to identify and  remove later.

Reply:
added comments on deprecated fields